src/app/(auth)/reset-password/page.tsx

```diff
@@ -0,0 +1,53 @@
+'use client';
+
+import { useState } from 'react';
+import { useRouter } from 'next/navigation';
+import TextInput from '@/components/TextInput/index';
+import { H1, H4 } from '@/styles/text';
+import { OuterDiv, FormDiv, QuestionsDiv } from '@/app/(auth)/styles';
+import BigButton from '@/components/BigButton';
+import supabase from '../../../api/supabase/createClient';
+
+export default function ResetPassword() {
+  const [newPassword, setNewPassword] = useState('');
+  const [newPassword2, setNewPassword2] = useState('');
```

Comment:
(2) Let's make a new boolean useState called `canReset` or something, and initialize it to false.